src/app/core/services/geo-position/geo-position.service.ts

```diff
@@ -34,7 +34,7 @@ const POSITION_OPTIONS_DEFAULT: PositionOptions = {
 
 const POSITION_OPTIONS_ANDROID: PositionOptions = {
   enableHighAccuracy: true,
-  timeout: 1000, //get notified with new position data at least each 1 sec
+  timeout: 20 * 1000, // 20 sek
```

Comment:
Har denne parameteren blitt brukt feil tidligere?

Reply:
Nja.. 
Jeg tror vi ønsker å få beskjed om posisjon ca. hver 20. sek. Tidligere var det slik at det kunne ta inntil 20 sekunder å få dette fra Android første gangen fra vi starter å abonnere på posisjonsdata. Derfor startet vi først å abonnere med timeout på 1 sek, og når vi fikk første posisjon, startet vi et nytt abonnement med timeout på 20 sek.
Dette hacket tror jeg forsvant på et eller annet tidspunkt, muligens i Angular-oppgraderingen, slik at vi nå fikk posisjon hvert sekund hele tida. 
Jeg så dette først nå og endret timeout til 20 sek for å spare ressurser. Har testet litt, og jeg har fått posisjon i løpet av et par sekunder første gangen nå, så jeg håper ikke vi trenger å legge inn hacket på nytt.